Gemfile

```diff
@@ -43,7 +43,7 @@ gem 'redis-namespace'
 
 gem 'active_model_serializers', '~> 0.8.3'
 
-gem 'onebox'
+gem 'onebox', git: 'https://github.com/vibol/onebox.git', branch: 'feature/amazon-instant-video'
```

Comment:
Ahh yes. Long story short, building an image from non-master didn't work for me. I'll resubmit the pull from a branch with screenshots and the specific commit.